Sources/Fingertips/FingerTipWindow.swift

```diff
@@ -233,16 +228,27 @@ public class FingerTipWindow: UIWindow {
             return
         }
 
-        UIView.animate(withDuration: fadeDuration) {
+        let animation = {
```

Comment:
The `animated` parameter wasn't being observed. As it is it seems that `animated` is always true, so we could simplify and rename to `removeFingerTipAnimated(with:)`.